src/randoop/util/SimpleList.java

```diff
@@ -40,11 +41,11 @@
  * creation time) proportional to N, not to the length of the new sequence.
  * <p>
  * When extending a Sequence with a new statement, we store the old sequence's
- * statements plus the new statement in a OneMoreElementList, which takes up
+ * statements plus the new statement in a {@code OneMoreElementList}, which takes up
  * only 2 references in memory (and constant creation time).
```

Comment:
"only 2 references": well, there's the object header overhead too.  No need to make any change, though.